internal/rbd/nodeserver.go

```diff
@@ -870,3 +887,27 @@ func openEncryptedDevice(ctx context.Context, volOptions *rbdVolume, devicePath
 
 	return mapperFilePath, nil
 }
+
+// xfsSupportsReflink checks if mkfs.xfs supports the "-m reflink=0|1"
+// argument. In case it is supported, return true.
+func (ns *NodeServer) xfsSupportsReflink() bool {
+	// return cached value, if set
+	if xfsHasReflink != xfsReflinkUnset {
+		return xfsHasReflink == xfsReflinkSupport
+	}
+
+	// run mkfs.xfs in the same namespace as formatting would be done in
+	// mountVolumeToStagePath()
+	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: utilexec.New()}
+	out, err := diskMounter.Exec.Command("mkfs.xfs").CombinedOutput()
+	if err != nil {
+		// mkfs.xfs should fail with an error message (and help text)
+		if strings.Contains(string(out), "reflink=0|1") {
+			xfsHasReflink = xfsReflinkSupport
+			return true
+		}
+	}
+
```

Comment:
@nixpanic do we need this whitespace here?

Reply:
Just clean coding, I think? This is not an else statement that would follow the if